components/about/Tweet.vue

```diff
@@ -43,20 +43,39 @@
 export default {
   props: {
     tweet: {
-      required: true,
       type: Object,
+      required: true
     },
+    turquoise: {
```

Comment:
instead of just `turqoise` should make it relevant to "why" it's turqoise.  If you really want turqoise, then should at least add a verb since it's a boolean. `isTurqoise` .

Reply:
I was attempting to follow the Vuetify style color naming here. I'm not 100% set on it tho.

Reply:
what naming do they have for prop names?

Reply:
it seems really strange that vuetify suggests naming booleans without verbs in them. That's typically a standard practice....I don't think we should always use vuetify as a truth for naming conventions and best practices 🤔